canvas_modules/common-canvas/src/common-properties/constants/form-constants.js

```diff
@@ -40,6 +40,12 @@ const Size = {
 	SMALL: "small"
 };
 
+const SingleLineLengthInSize = {
```

Comment:
instead of hardcoding these values can't we find the width of common-properties and calculate this?  common-properties allows for custom sizes so we don't always have small/medium/large